illink.sln

```diff
@@ -21,15 +19,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "analyzer", "analyzer", "{AA
 EndProject
```

Comment:
Could we remove the analyzer directory entry?

Reply:
Similar to tests - this is a supporting tool, not the product (or tests) so I personally like it in a separate subfolder. But if most people don't want it there, sure...